### PR TITLE
MM-26071 Cypress/E2E: Upgrade Cypress to 4.8.0 and other dependencies to latest

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Upgrade Cypress to 4.8.0 and other dependencies to latest

#### Ticket
Jira ticket - https://mattermost.atlassian.net/browse/MM-26071

Test results on CI:
- https://mm-cypress-report.s3.amazonaws.com/387-master-master/mochawesome.html
- Quick stats
| Passing Rate | 99.46% |
| Duration | 44.64 mins |
| Suites | 162 |
| Tests | 556 |
| :white_check_mark: Passed | 553 |
| :x: Failed | 3 |
| :fast_forward: Skipped | 0 |

### CODE REVIEW

File: e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js [hunk 1]
```diff
@@ -27,13 +27,13 @@ describe('Messaging', () => {
         cy.apiSavePreviewCollapsedPreference('false');
     });
 
-    it('M16425 : Show single image thumbnails in standard mode', () => {
+    it.only('M16425 : Show single image thumbnails in standard mode', () => {
```
Comment (josephbaylon) [inline e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js:30]: Remove `.only`

File: e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js [hunk 2]
```diff
@@ -59,13 +59,13 @@ describe('Messaging', () => {
         });
     });
 
-    it('M16425 : Show single image thumbnails in compact mode', () => {
+    it.only('M16425 : Show single image thumbnails in compact mode', () => {
```
Comment (josephbaylon) [inline e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js:62]: Remove `.only`

File: e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/messaging/file_upload_in_center_channel_spec.js]: Thanks for catching. Should be good now.

**Comment by srkgupta** [comment]
@saturninoabril 
Can you please fix the merge conflict now that the other PR was merged.

**Pull request merged**